flatgeobuf/flatgeobuf.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Why do we need to download and save it? To emphasize reading from the cloud, I would skip this download and store step if at all possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Maybe it makes sense to reverse the order of the sections so that reading from the cloud happens before reading and writing locally?

I think I set it up this way because the cloud-native vector ecosystem is just younger and doesn't work as seamlessly as the raster/zarr sides. Like the remote fetch of vector features in my remote example took 2 minutes... I imagine some users would try to run that cell (on a slower internet connection than mine) and assume something went wrong.

Reply:
I see 🤔 well if its possible to bypass downloading to get your points across, that would be preferable, but if not, perhaps we can just add a disclaimer similar to what you just wrote:

>The cloud-native vector ecosystem is younger and doesn't work as seamlessly as the COG and Zarr for subsetted access. We download data here to demonstrate important concepts but look to update these methods once better datasets and tools are available for working with flatgeobuf without downloading entire files.